src/routes/products/products.controller.ts

```diff
@@ -306,24 +306,30 @@ router.get(
           minPrice
         )}, to: ${Number(maxPrice)}})
         .pageSize(${Number(pageSize)}) {
+          id,
           name,
           description,
           price,
```

Comment:
Minor nit, but `price` and `description` are in a different order in `GET /products` responses.